app/workers/import_worker.rb

```diff
@@ -0,0 +1,15 @@
+class ImportWorker
+  include Sidekiq::Worker
+
+  def perform(device, json_file)
```

Comment:
I'll try this but I think had some problems with the path no longer being valid unless the worker started right away sometimes.